README.md

```diff
@@ -73,6 +76,25 @@ This command is deprecated in favor of `ember try:one`. There are several bugs w
 
 ### Config
 
+##### versionCompatibility
+If you're using `ember-try` with an Ember addon, there is a short cut to test many Ember versions. In your `package.json` under the `ember-addon` key, add the following: 
+
+```json
+  "ember-addon": {
+    "versionCompatibility": {
+       "ember": ">1.11.0 <=2.0.0"
```

Comment:
We chatted on slack but commenting here to document some of what we said:

"beta" and "canary" scenarios get added to whatever is generated by this statement. I like this because it requires addons to test against those channels. Those scenarios can be overridden by creating an `ember-try.js` that creates scenarios with the same name (run `ember try:config` to see what the config would have been).